tiddlers/stylesheet.tid

```diff
@@ -13,24 +12,33 @@ ul.checklist > li {
 }
 
 ul.checklist > li:first-child {
-    margin-bottom: 2px;
+    margin-bottom: 0.75em;
 }
 
-span.checklistitem-body {
-    margin-left: 5px;
+label.checklistitem-body {
+    margin-left: -0.9em;
+    margin-right: 3em;
+    position: relative;
+    left: 1.75em;
+}
+
+input:checked + label.checklistitem-body {
+    color: <<colour muted-foreground>>;
 }
 
-span.checkedchecklistitem-body {
-    color: #ddd;
+<$list filter="[[$:/plugins/tgrosinger/tw5-checklist/Configuration]field:strike-checked[true]]">
```

Comment:
This looks really cool, can you put a comment on the line above explaining what is happening?